modules/status-roles/events/presenceUpdate.js

```diff
@@ -0,0 +1,22 @@
+const {localize} = require("../../../src/functions/localize");
+
+module.exports.run = async function (client, newPresence) {
+
+    if (!client.botReadyAt) return;
+    const moduleConfig = client.configurations['status-roles']['config'];
```

Comment:
Please check if the member is on the guild the bot runs on first. 
Example: `if (newPresence.member.guild.id !== client.guildID) return`